ckanext/tabledesigner/datastore.py

```diff
@@ -50,9 +51,7 @@ def create_table(resource_id: str, fields: List[dict[str, Any]]):
             validate_rules=''.join(validate_rules),
         )
         get_action('datastore_function_create')(
-            {
-                'ignore_auth': True,
-            },
```

Comment:
This one should be fine, no? `create_table` is a function for internal use, not tied to a user or session.